includes/REST/ProductController.php

```diff
@@ -275,6 +275,10 @@ public function validation_before_create_item( $request ) {
             return new WP_Error( 'no_store_found', __( 'No seller found', 'dokan-lite' ), [ 'status' => 404 ] );
         }
 
+        if ( ! dokan_is_seller_enabled( $store_id ) ) {
+            return new WP_Error( 'invalid_request', __( 'Error! Your account is not enabled for selling, please contact the admin', 'dokan-lite' ), [ 'status' => 400 ] );
```

Comment:
change the message -  add `with` before `the admin`
`Error! Your account is not enabled for selling, please contact with the admin`

Reply:
@ManiruzzamanAkash `contact with the admin` is not the correct English wording.
It should be like it was before: `please contact the admin`